checker/tests/nullness/MisuseProperties.java

```diff
@@ -16,11 +16,11 @@ void propertiesToHashtable(Properties p) {
     p.put("line.separator", null);
     Hashtable h = p;
     // Error, because HashTable value has NonNull bound.
-    // TODO: false negative. See #365.
     //// :: error: (argument) :: warning: [unchecked] unchecked call to
     //// put(K,V) as a member of the raw type java.util.Hashtable
```

Comment:
Should these two lines be deleted?  They are redundant with the following three lines.